mithril-common/src/entities/type_alias.rs

```diff
@@ -20,3 +20,6 @@ pub type MagicId = u64;
 
 /// Protocol version
 pub type ProtocolVersion = String;
+
+/// Hex encoded key
+pub type HexEncodedKey = String;
```

Comment:
In relation with the comment above, it might make sense to add types for vk sig and opcert, and use it above. 
```suggestion

/// Hex encoded key
pub type HexEncodedKey = String;

/// Hex encoded Verification Key Signature
pub type HexVkSignature = String;

/// Hex encoded OpCert
pub type HexOpCert = String;
```

Reply:
Done :heavy_check_mark: